extensions/amp-a4a/rtc-documentation.md

```diff
@@ -67,7 +67,8 @@ The value of `rtc-config` must conform to the following specification:
 	},
     "urls": [
         "https://www.exampleA.com/endpoint",
-        "https://www.exampleb.com/endpoint"],
+        {"url": "https://www.exampleb.com/endpoint",
+          "errorReportingUrl":"https://www.exampleb.com/endpoint?e=ERROR_TYPE&h=HREF"}],
```

Comment:
Nit: capitalize exampleB, to match line 69?

Reply:
done